src/shared/crypto/bitcoin/signer-config.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { HDKey } from '@scure/bip32';
+import { mnemonicToSeedSync } from '@scure/bip39';
+import * as btc from '@scure/btc-signer';
```

Comment:
How is this working here? 🤔 Why do I run into issues with importing btc-signer in tests? Maybe now fixed with the playwright upgrade?